### PR TITLE
chore(color) - LVGL update and translation strings for theme file carousel.

### PR DESCRIPTION
**Pull request by philmoz**
Use StaticText instead of 'paint' function for 'loading' message in theme file carousel.
Add message strings to translation files.

### CODE REVIEW

**Comment by pfeerick** [comment]
Can we get some translations for this please 😃 

image


```
#define TR_NO_THEME_IMAGE "No theme image"
```

- [ ] @HThuren DA
- [ ] @ParkerEde / @TheIsotopes DE
- [ ] @Eldenroot CZ
- [ ] @ulfhedlund SE
- [ ] @robustini IT
- [ ] @zyren CN / TW
- [ ] @ajjjjjjjj PL
- [ ] @ToshihiroMakuuchi JP

**Comment by HThuren** [comment]
DA
#define TR_EDIT_COLOR                  "Rediger farve"
#define TR_NO_THEME_IMAGE              "Ingen billede for tema"

**Comment by robustini** [comment]
IT
#define TR_EDIT_COLOR           "Edita colore"
#define TR_NO_THEME_IMAGE "Nessuna\nimmagine\ntrovata"

I hope there are three lines available.
Italian is a wonderful language but it is difficult to abbreviate words.

**Comment by philmoz** [comment]
> #define TR_NO_THEME_IMAGE "Nessuna\nimmagine\ntrovata"
> 
> I hope there are three lines available. Italian is a wonderful language but it is difficult to abbreviate words.

Thanks, forced newlines are not necessary. The text will wrap as needed.

**Comment by ulfhedlund** [comment]
SE: #define TR_NO_THEME_IMAGE "Ingen temabild"Den 1 mars 2023 00:42 skrev Peter Feerick ***@***.***>:

Can we get some translations for this please 😃



#define TR_NO_THEME_IMAGE "No theme image"



 @HThuren DA @ParkerEde / @TheIsotopes DE @Eldenroot CZ @ulfhedlund SE @robustini IT @zyren CN / TW @ajjjjjjjj PL @ToshihiroMakuuchi JP

**Comment by TheIsotopes** [comment]
DE

```
#define TR_EDIT_COLOR                  "Farbe bearbeiten"
#define TR_NO_THEME_IMAGE              "Kein Theme Bild"
```

File: radio/src/translations/cz.h [hunk 1]
```diff
@@ -743,6 +743,7 @@
 #define TR_DELETE_THEME                "Smazat motiv?"
 #define TR_SAVE_THEME                  "Uložit motiv?"
 #define TR_EDIT_COLOR                  "Edit Color"
```
Comment (Eldenroot) [inline radio/src/translations/cz.h:745]: Změnit barvu

File: radio/src/translations/cz.h [hunk 2]
```diff
@@ -743,6 +743,7 @@
...
+#define TR_NO_THEME_IMAGE              "No theme image"
```
Comment (Eldenroot) [inline radio/src/translations/cz.h:746]: Náhled motivu nedostupný

**Review by Eldenroot — CHANGES_REQUESTED** [review]

**Comment by Eldenroot** [comment]
CZ commented

**Comment by ToshihiroMakuuchi** [comment]
JP
#define TR_EDIT_COLOR "カラー編集"
#define TR_NO_THEME_IMAGE "テーマ画像はありません"

**Comment by zyren** [comment]
********************************************************
CN

#define TR_EDIT_COLOR "编辑颜色"
#define TR_NO_THEME_IMAGE "没有预览图"
*********************************************************
*********************************************************
TW

#define TR_EDIT_COLOR "編輯顏色"
#define TR_NO_THEME_IMAGE "沒有預覽圖"
*********************************************************

**Comment by Eldenroot** [comment]
CZ: "Upravit barvu"

This fits better

@philmoz

**Pull request merged**